Gemfile

```diff
@@ -38,7 +38,8 @@ gem 'mail'
 gem 'mime-types', require: 'mime/types/columnar'
 
 gem 'hiredis'
-gem 'redis', require:  ["redis", "redis/connection/hiredis"]
+# https://github.com/redis/redis-rb/pull/591
+gem 'redis', require:  ["redis", "redis/connection/hiredis"], github: 'tgxworld/redis-rb'
```

Comment:
let's freedom patch this the test will cover that it is still working on redis upgrades. we just need to remember to remove the freedom patch so add a note there.